go.mod

```
@@ -0,0 +1,8 @@
+module bootCampApi
```

Module name should be the same as your path in your code repository (in this case github).

```suggestion
module github.com/jcaladwizeline/academy-go-q42021
```